contracts/context-config/src/lib.rs

```diff
@@ -17,12 +16,15 @@ mod sys;
 use guard::Guard;
 
 const DEFAULT_VALIDITY_THRESHOLD_MS: Timestamp = 10_000;
+const DEFAULT_CONTRACT: &[u8] = include_bytes!("../../proxy-lib/res/proxy_lib.wasm");
 
 #[derive(Debug)]
 #[near(contract_state)]
 pub struct ContextConfigs {
     contexts: IterableMap<ContextId, Context>,
     config: Config,
+    proxy_code: LazyOption<Vec<u8>>,
```

Comment:
should there ever be a case where it's `None`?

if not, then maybe we use `Vector<u8>`?